paper-icon-button.html

```diff
@@ -135,6 +139,21 @@
        */
       icon: {
         type: String
+      },
+
+      /**
+       * Specifies the alternate text for the button, for accessibility.
+       */
+      alt: {
+        type: String,
+        observer: "_altChanged"
+      }
+    },
+
+    _altChanged: function() {
+      var label = this.getAttribute('aria-label');
+      if (!label) {
```

Comment:
Maybe have a flag to say whether `aria-label` was set by the author (not sure about terminology - what do you call a web page author using a custom element?) or by the element, and update when/if the author sets the `aria-label` attribute? Is that feasible?